arm-resources/locks/2016-09-01/swagger/locks.json

```diff
@@ -186,7 +186,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The scope for the lock. "
+            "description": "The scope for the lock. A scope can be defined at subscription, resource group, or resource level. It looks like this: /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{parentResourcePathIfPresent}/{resourceType}/{resourceName}"
```

Comment:
@mcardosos  Yes, it'd be good to have a more complete description, as it may get picked up for documentation, and this way users have more details.